ExcelMapper/ExcelMapper.cs

```diff
@@ -151,6 +152,18 @@ public IEnumerable Fetch(string file, Type type, string sheetName)
             return Fetch(type, sheetName);
         }
 
+        /// <summary>
+        /// Fetches objects from the specified sheet name.
+        /// </summary>
+        /// <param name="file">The path to the Excel file.</param>
+        /// <param name="sheetName">Name of the sheet.</param>
+        /// <returns>The objects read from the Excel file.</returns>
+        public IEnumerable<dynamic> FetchDynamic(string file, string sheetName)
```

Comment:
Can't we overload `Fetch` so we don't need `FetchDynamic`?

Reply:
I don't think you can. 
`Fetch<T>()` means you work with a `Type T` where T _is defined_.
You cannot overload with `Fetch<dynamic>()`. Generics imply a defined `Type` at compile time.

You can't use the existing Fetch() because `typeof(dynamic)` is not allowed.

`CS1962 | Error | The typeof operator cannot be used on the dynamic type`

Reply:
because of the peculiar nature of dynamic trying to reduce the amount of method will result in an overly complex IEnumerable Fetch(ISheet sheet, Type type) with param type == null.